packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -106,20 +106,27 @@ async function withCustomPersistence(
     PlatformSupport.getPlatform(),
     new SharedFakeWebStorage()
   );
-  const multiClientParams = settings.experimentalTabSynchronization
-    ? {
-        sequenceNumberSyncer: MOCK_SEQUENCE_NUMBER_SYNCER
-      }
-    : undefined;
-
-  const persistence = new IndexedDbPersistence(
-    TEST_PERSISTENCE_PREFIX,
-    clientId,
-    platform,
-    queue,
-    serializer,
-    multiClientParams
-  );
+  if (settings.experimentalTabSynchronization) {
+  } else {
+  }
```

Comment:
Hm, I didn't see you add a test for this part of the PR :)

Reply:
Obviously, it's trivially bug-free.

But since I don't want to write a test for it, I'll delete it.